packages/fast-tooling-react/src/navigation/navigation.style.ts

```diff
@@ -101,12 +119,21 @@ const styles: ComponentStyles<NavigationClassNameContract, {}> = {
         cursor: "inherit",
         textDecoration: "none",
         fontStyle: "italic",
-        "&:focus": {
```

Comment:
I think we still want focus. Also should the expand collapse arrows have focus?

Reply:
Added back in focus visible, the expand buttons shouldn't get focus (i believe)